exercises/matrix/MatrixTest.fs

```diff
@@ -32,8 +32,8 @@ let ``Can extract column`` () =
     column 3 "1 2 3\n4 5 6\n7 8 9" |> should equal [3; 6; 9]
 
 [<Fact(Skip = "Remove to run test")>]
-let ``Can extract column from non-square matrix`` () =
-    column 3 "1 2 3\n4 5 6\n7 8 9\n8 7 6" |> should equal [3; 6; 9; 6]
+let ``Can extract column from non-square matrix with no corresponding row`` () =
+    column 4 "1 2 3 4\n 5 6 7 8\n 9 8 7 6" |> should equal [4; 8; 6]
```

Comment:
I think the problem is the extra space after the 4\n part. Looks like a possible problem with the canonical data.

Reply:
Yup looks like it was a typo:
https://github.com/exercism/problem-specifications/pull/1578/files